charts/all-in-one/templates/service.yaml

```
@@ -759,3 +759,21 @@ spec:
     app: rudolf-db
   type: ClusterIP
 {{ end }}
```

It needs separator to separate yaml documents.

```suggestion
  type: ClusterIP
---
{{ end }}
```